src/appleseed-max-impl/appleseeddisneymtl/appleseeddisneymtl.cpp

```diff
@@ -168,6 +169,33 @@ namespace
         ParamIdBumpTexmap
     };
 
+    void update_map_buttons(IParamBlock2* pblock)
```

Comment:
Do we really need this function _and_ the `TexturePBAccessor` class?

Reply:
Agree it's crazy. I think I can get rid of accessor and put the same code into setsubtexmap function.